### ISSUE TITLE
[ams 0.10.0.rc2] ArgumentError - Unknown adapter: json_api

### ISSUE THREAD

**Issue by piotrpalek** [rails-api/active_model_serializers#993]
Hey, I am adding `ActiveModel::Serializer.config.adapter = :json_api` in `app/config/initializers/active_model_serializers.rb` and when I try to fetch some data I get this error: `ArgumentError - Unknown adapter: json_api. Valid adapters are: [":fragmentcache", ":json", ":flattenjson", ":null", ":jsonapi"]`

**Comment by wonderbread** [comment]
Same issue.

Work around: 
```ruby
# config/initializers/active_model_serializers.rb
ActiveModel::Serializer.config.adapter = ActiveModel::Serializer::Adapter::JsonApi
```

My reason (likely yours too?):
```ruby
# config/initializers/inflections.rb
ActiveSupport::Inflector.inflections(:en) do |inflect|
  inflect.acronym 'API'
end
```
`.classify` in [this method](rails-api/active_model_serializers/blob/90fb1cf809dfa606181a1e047f30a05efe71912a/lib/active_model/serializer/adapter.rb#L35-L37) is turning it into `jsonAPI` instead of `jsonApi`

**Comment by piotrpalek** [comment]
Good catch, yeah the workaround you posted works, forgot to update the issue with it. Nice catch with the inflector :+1:

**Comment by bf4** [comment]
Do you think you can write a failing test and/or a PR to fix this?

**Comment by joaomdmoura** [comment]
tks you all! great work! Just flaged the PR as good for new contributors.

**Comment by bf4** [comment]
Fixed by #997  and https://github.com/rails-api/active_model_serializers/pull/1006
